gapic-generator/templates/default/service/test/method/_normal.erb

```diff
@@ -65,7 +65,7 @@ def test_<%= method.name %>
     end
 
     # Use protobuf object with options
-    client.<%= method.name %> <%= method.request_type %>.new(<%= fields.map(&:as_kwarg).join ", " %>), grpc_options do |response, operation|
+    client.<%= method.name %>(<%= method.request_type %>.new(<%= fields.map(&:as_kwarg).join ", " %>), grpc_options) do |response, operation|
```

Comment:
more braces!

Reply:
Yeah, well, there turned out to be a few cases where we could be generating ambiguous code otherwise.